### DISCUSSION TITLE
Successfully fixed my iPhone 5 that wouldn't boot due to a full disk!

### DISCUSSION THREAD

**Discussion by badger200** [LukeZGD/Legacy-iOS-Kit#583] [General]
Wow man thanks a million for this tool! I couldn't even find any tools that implemented 32 bit checkm8, with the author merely saying it's theoretically possible... When here you've got it all rolled up into one sweet solution!

I've been waiting like 4 years for a tool like this to appear, I'm so happy, I kept faith someone would some day make such a tool and you did it!

I used ssh ramdisk to boot and fsck my disk, that was enough to fix my iPhone 5 that would just freeze on the white Apple logo! It booted iOS and runs properly now for the first time in 4 years!

I'm also using this to attempt data recovery from my iPad Pro 9.7" which has a dead NAND from too many writes (long story). I'm hoping I can mount the data partition	on read-only but so far it's failed. I'll make a separate Issue for that someday.

**Comment by LukeZGD** [comment]
np, its great that this project is helping out a lot of people in many ways
